src/domain/model/authentication/Roles.ts

```diff
@@ -0,0 +1,3 @@
+export const USER = 'USER';
+export const DOCTOR = 'DOCTOR';
```

Comment:
Is doctor the right name?  It could be a pharmacist, nurse or other trusted person.  They could be different roles of course.

Reply:
Yeah.. I am not sure I have the right roles here at all.. Let's re-visit this once we know..